internal/querycoordv2/job/scheduler.go

```diff
@@ -63,6 +63,9 @@ func (scheduler *Scheduler) Stop() {
 	scheduler.stopOnce.Do(func() {
 		close(scheduler.stopCh)
 		scheduler.wg.Wait()
+		for _, queue := range scheduler.queues {
```

Comment:
it should be closed in schedule routine.
Otherwise scheduler might crash

Reply:
The `scheduler.wg.Wait()` will make sure that the go-routine has finished.
https://github.com/milvus-io/milvus/blob/c5cd92d36eb9ea35dd7cfd25fa364eaddfd4b1c1/internal/querycoordv2/job/scheduler.go#L120-L143